pom.xml

```diff
@@ -54,12 +54,14 @@
     <jenkins-core.version>${jenkins.version}</jenkins-core.version>
     <jenkins-war.version>${jenkins.version}</jenkins-war.version>
     <jenkins-test-harness.version>2.44</jenkins-test-harness.version>
-    <hpi-plugin.version>2.7</hpi-plugin.version>
+    <hpi-plugin.version>2.8-20181123.072741-1</hpi-plugin.version>
     <stapler-plugin.version>1.17</stapler-plugin.version>
     <java.level>you-must-override-the-java.level-property</java.level>
+    <plugin.minimumJavaVersion>1.${java.level}</plugin.minimumJavaVersion>
```

Comment:
I feel something is a bit wrong here. This is probably time to bite the bullet and use `{java.level}` instead of `1.${java.level}`?

Reply:
It has a higher risk of compatibility regressions than the current approach

Reply:
The problem is that Java 11 is not `1.11` but `11`. So it will be confusing for developers at some point.

Reply:
Yes, but it is `1.8` for Java 8 which stays a default version in this Plugin POM for foreseeable future. Java 11 is supported via the profile

Reply:
I agree. I believe there is no better solution here.